src/test/rustdoc-gui/type-declation-overflow.goml

```diff
@@ -5,4 +5,4 @@ size: (1100, 800)
 // Logically, the <body> scroll width should be the width of the window.
 assert-property: ("body", {"scrollWidth": "1100"})
 // However, since there is overflow in the type declaration, its scroll width is bigger.
-assert-property: (".type-decl pre", {"scrollWidth": "1324"})
+assert-property: (".type-decl pre", {"scrollWidth": "1332"})
```

Comment:
Normally, this shouldn't have changed...

Reply:
I reverted the change to the test and instead changed CSS to remove padding from `<pre><code>` blocks.